client/src/components/ChatMessage.vue

```diff
@@ -103,6 +118,12 @@ export default {
       );
       return '';
     },
+    messageStatus() {
+      if (this.$props.message.seen?.length > 1) return 'seen';
```

Comment:
to improve for group chats, ideally see the list of people who have seen the message